save.py

```diff
@@ -10,12 +10,13 @@
 
 import aiohttp_jinja2
 
-from nameinternal import get_card, get_card_metadata
+from nameinternal import get_card
 from sts_profile import get_current_profile
 from typehints import ContextType
 from gamedata import FileParser, BottleRelic
 from webpage import router
 from logger import logger
+from score import score as _s
```

Comment:
I don't think we need to have a `score` package; the file probably should just be `score.py` rather than `score/score.py`